07_inf_for_numerical_data/inf_for_numerical_data.Rmd

```diff
@@ -13,20 +13,25 @@ output:
 knitr::opts_chunk$set(eval = FALSE)
 library(tidyverse)
 library(oilabs)
+library(infer)
+library(openintro)
 ```
 
 ## Getting Started
 
 ### Load packages
 
 In this lab, we will explore and visualize the data using the `tidyverse` suite of 
-packages. The data can be found in the companion package for OpenIntro labs, `oilabs`.
+packages, and perform statistical inference using `infer`. The data can be found
+in the companion package for OpenIntro labs, `oilabs`.
```

Comment:
Actually we're moving everything into the `openintro` package so you can update the language as such